test/worlds/gpu_lidar.sdf

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" ?>
+<!--
+  Note: For now, this file must be loaded from ign-gazebo's root folder so the
+  mesh's relative path is correct.
```

Comment:
This comment seems out of place

Reply:
Ah, copied from the other gpu world file. I'll remove them both.

Reply:
Removed in f32131b